### DISCUSSION TITLE
Use DLNA on the devices’ subnet instead of server’s subnet

### DISCUSSION THREAD

**Discussion by LoMGolden** [jellyfin/jellyfin#4914] [General] [answered]
Hello everyone !

I don’t know if it’s possible but I think it could be a good idea of using DLNA not on the devices connected to the server subnetwork but on the device using Jellyfin. (So if there’s multiple devices using Jellyfin from different places, each Jellyfin user will see its own available devices).
It could be really useful if you have family or friends using your server, it permits them to use DLNA in their home !

Thanks a lot for all your work again 💯 !

**Comment by anthonylavado** [comment, marked as answer]
DLNA works as a server > client system. To do this, it would mean adding DLNA serving to each client app, which isn't practical for Jellyfin. The way the DLNA works was architected around the idea of having a server in your own home, that serves to client devices there.

So this is, unfortunately, not possible.

**Reply by LoMGolden** [reply]
Okk I understand, thanks for the time you took :)
